res/res_pjsip_sdp_rtp.c

```diff
@@ -2246,8 +2246,10 @@ static int apply_negotiated_sdp_stream(struct ast_sip_session *session,
 		ast_rtp_instance_set_timeout(session_media->rtp, session->endpoint->media.rtp.timeout_hold);
 	}
 
-	if (ast_rtp_instance_get_timeout(session_media->rtp)) {
-		session_media->timeout_sched_id = ast_sched_add_variable(sched,	500, rtp_check_timeout,
+	int rtp_timeout = ast_rtp_instance_get_timeout(session_media->rtp);
```

Comment:
This will not compile under dev mode (when developing you should always build using "./configure --enable-dev-mode" which will turn warnings into errors and stop builds).  Variables need to be declared at the start of the function.